datadog_checks_base/datadog_checks/base/checks/db.py

```diff
@@ -0,0 +1,43 @@
+# (C) Datadog, Inc. 2025-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+
+from datadog_checks_base.datadog_checks.base.checks.base import AgentCheck
+
+
+class DBCheck(AgentCheck):
+    def database_monitoring_query_sample(self, raw_event):
+        # type: (str) -> None
```

Comment:
We can use the proper type checking instead of the py2 version of it. We will eventually remove all this, we have a story for that.